streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamThread.java

```diff
@@ -400,17 +400,21 @@ private RuntimeException unAssignChangeLogPartitions() {
 
     @SuppressWarnings("ThrowableNotThrown")
     private void shutdownTasksAndState() {
-        log.debug("{} shutdownTasksAndState: shutting down all active tasks {} and standby tasks {}", logPrefix,
-            activeTasks.keySet(), standbyTasks.keySet());
+        log.debug("{} shutdownTasksAndState: shutting down all active tasks {} " +
+                "and standby tasks {} and suspended tasks {} and suspended standby tasks {}", logPrefix,
```

Comment:
nit: "active tasks {}, standby tasks {}, suspended tasks {}, and suspended standby tasks {}"